test.js

```diff
@@ -210,6 +210,28 @@ t.test('options.callbackUri should be and object', t => {
     })
 })
 
+t.test('options.callbackUriParams should be and object', t => {
+  t.plan(1)
+
+  const fastify = createFastify({ logger: { level: 'silent' } })
+
+  fastify.register(oauthPlugin, {
+    name: 'the-name',
+    credentials: {
+      client: {
+        id: 'my-client-id',
+        secret: 'my-secret'
+      },
+      auth: oauthPlugin.GITHUB_CONFIGURATION
+    },
+    callbackUri: '/callback',
+    callbackUriParams: 1
```

Comment:
There is no test for this value. Could you invoke `startRedirectHandler` in order to check the redirection has this value in the query string?

Reply:
Hi, please help me check if i did right 💯 .

Reply:
Friendly ping to @allevo since @meotimdihia pushed another commit including the test